src/app/shared/components/autocomplete/autocomplete.component.spec.ts

```diff
@@ -0,0 +1,66 @@
+import {Component, CUSTOM_ELEMENTS_SCHEMA, DebugElement} from '@angular/core';
+import { ComponentFixture, TestBed, fakeAsync, tick } from '@angular/core/testing';
+import { FormControl, FormsModule, NgForm, ReactiveFormsModule } from '@angular/forms';
+import { AutocompleteComponent } from './autocomplete.component';
+import {By} from '@angular/platform-browser';
+
+describe('AutocompleteComponent', () => {
+  let component: TestAutocompleteComponent;
```

Comment:
Can we test on the AutocompleteComponent instead?
Why are we creating this TestAutocompleteComponent if it is not used ?

Reply:
TestAutocompleteComponent is a mock I used to test the behaviour of AutocompleteComponent, we can discuss about it.